prow/config.yaml

```diff
@@ -13921,6 +14098,34 @@ periodics:
       secret:
         secretName: velodrome-influxdb
 
+- name: periodic-kubernetes-bazel-build-1-10
+  interval: 6h
+  agent: kubernetes
+  labels:
+    preset-service-account: true
+    preset-bazel-scratch-dir: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20180226-a50c3d6d9-1.10
+      args:
+      - "--clean"
+      - "--job=$(JOB_NAME)"
+      - "--repo=k8s.io/kubernetes=release-1.10"
+      - "--service-account=/etc/service-account/service-account.json"
+      - "--upload=gs://kubernetes-jenkins/logs"
+      - "--" # end bootstrap args, scenario args below
+      - "--build=//... -//vendor/..."
+      - "--release=//build/release-tars"
+      env:
```

Comment:
one last fix, for the *bazel* periodics (not postsubmit) can you set `env` `REPO_NAME` = `kubernetes` `REPO_OWNER` = `kubernetes`? The caching leverages these prow env to select which cache based on primary workspace, but the periodics don't have this and the existing fallback doesn't work. I'm working on better options but those will take a bit longer. issue: #7023

Reply:
I see you got this for the test job eblow, but not the build here.

Reply:
Fixed.